bin/serverless

```diff
@@ -2,26 +2,32 @@
 
 'use strict';
 
+const autocomplete = require('../lib/utils/autocomplete');
 const BbPromise = require('bluebird');
 const logError = require('../lib/classes/Error').logError;
 
-BbPromise.config({
-  longStackTraces: true,
-});
+if (process.argv[2] === 'completion') {
+  autocomplete();
+} else {
+  BbPromise.config({
```

Comment:
@eahefnawy it may be worth moving this code to above the `if` check. I think we still want long stack traces and error tracking on autocomplete. That way we can detect if there is a problem with the autocomplete. 
```js
BbPromise.config({
 longStackTraces: true,
});
process.on('unhandledRejection', (e) => logError(e));
process.noDeprecation = true;
```